test/selenium/lib/foxboxHelperSelenium.js

```diff
@@ -4,15 +4,16 @@ const path = require('path');
 
 const spawn = require('child_process').spawn;
 
+const PROFILE_PATH = path.join(process.env.HOME, '.local/share/foxbox-tests');
```

Comment:
Added the trailing slash (the path wasn't correct, otherwise). It seems either the sign up database is not removed or we don't contact the right foxbox.